### DISCUSSION TITLE
Where to set the EMIntuneAppPackages setting

### DISCUSSION THREAD

**Discussion by stefan968** [Micke-K/IntuneManagement#58] [Q&A] [answered]
Where can I set the EMIntuneAppPackages setting in the GUI? I modified the EndpointManager.psm1 file to add a default value which works but I was wondering what a proper way is to configure this setting?

**Comment by Micke-K** [comment, marked as answer]
In Settings (File -> Settings), and then specify the path in the "App packages folder" setting

![image](https://user-images.githubusercontent.com/43226266/175044990-e15175a2-faf6-4918-974b-df7ac7ab395e.png)

**Comment by stefan968** [comment]
Oh wow thanks. i completely overlooked that.
